### PR TITLE
Add policies policyset

### PR DESCRIPTION
**Pull request by RobDolares**
Related Issue: https://github.com/stolostron/backlog/issues/19215

Successful states:
<kbd>Screen Shot 2022-03-16 at 4 48 21 PM</kbd>
<kbd>Screen Shot 2022-03-16 at 4 48 30 PM</kbd>

Alert from selecting policies from different namespaces:
<kbd>Screen Shot 2022-03-16 at 4 48 45 PM</kbd>

Alert when no policy set exists in namespace of selected policy:
<kbd>Screen Shot 2022-03-16 at 4 49 15 PM</kbd>

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -39,6 +39,8 @@
     "Add": "Add",
     "Add credential": "Add credential",
     "Add host": "Add host",
+    "Add to policy set": "Add to policy set",
+    "Add to Policy set": "Add to Policy set",
```
Comment (zlayne) [inline frontend/public/locales/en/translation.json:43]: duplicate

File: frontend/src/routes/Governance/policies/Policies.tsx [hunk 2]
```diff
@@ -621,6 +644,155 @@ function usePolicyViolationsColumn(
...
+
+    useEffect(() => {
+        setSelectedPolicySet(namespacedPolicySets.find((ps) => ps.metadata.uid === selectedPolicySetUid))
+    }, [selectedPolicySetUid, namespacedPolicySets])
+
+    const [error, setError] = useState('')
+    const onConfirm = async () => {
+        setIsAdding(true)
+        try {
+            setError('')
```
Comment (zlayne) [inline frontend/src/routes/Governance/policies/Policies.tsx:667]: Do you need this here? Is it for trying to submit again after error?

**Review by zlayne — APPROVED** [review]
/hold

**Review by zlayne — APPROVED** [review]

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**